src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/Globals.vb

```diff
@@ -7,4 +7,5 @@ Namespace Global.Microsoft.VisualBasic
         [Binary] = 0
         [Text] = 1
     End Enum
+
```

Comment:
Change can be reverted.

Reply:
Reverted to what, this is in reference source line 54-57